src/hotspot/share/gc/shared/preservedMarks.hpp

```diff
@@ -34,20 +34,21 @@ class WorkerTask;
 class PreservedMarksSet;
 class WorkerThreads;
 
+class OopAndMarkWord {
```

Comment:
Now that this class is in public domain, I tend to think it should be renamed to `PreservedMark` to be consistent with other classes in this file. (Ofc, this is quite subjective.)

Reply:
Renamed.